composer.json

```
@@ -9,12 +9,12 @@
         }
     ],
     "require": {
-        "php": "~7.1",
+        "php": "~7.2",
         "guzzlehttp/guzzle": "~6.0"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "~2.8",
-        "phpunit/phpunit": "~6.0",
+        "phpunit/phpunit": "^8.0",
```

Can we support both phpunit versions?
```suggestion
        "phpunit/phpunit": "~6.0||^8.0",
```